### ISSUE TITLE
ORKInstructionStepView is always setting alignment to left with attributed string

### ISSUE THREAD

**Issue by sainianil** [ResearchKit/ResearchKit#1230]
ORKInstructionStepView is always setting alignment to left instead of provided alignment as an attributed string.

**Comment by srinathtm-apple** [comment]
@sainianil - have you tried leveraging the attributedDetailText property of ORKInstructionStep?

**Comment by sainianil** [comment]
Yes, I tried it. However, I didn't set any alignment so as per my understanding alignment should be set to NSTextAlignmentNatural by default.

<img width="776" alt="screenshot 2019-02-27 at 6 30 43 pm" src="https://user-images.githubusercontent.com/10446286/53492176-3a6e1180-3abe-11e9-99eb-c677e70530ca.png">
